cime_config/namelist_definition_blom.xml

```diff
@@ -3968,6 +3968,16 @@
     <desc>Logical switch to activate riverine input</desc>
   </entry>
 
+    <entry id="use_r2o">
+    <type>logical</type>
+    <category>bgcnml</category>
+    <group>bgcnml</group>
+    <values>
+      <value>.false.</value>
+    </values>
+    <desc>Logical switch to use riverine carbon and nutrient inputs following R2OMIP protocol</desc>
+  </entry>
+
```

Comment:
To enable switching on the output (once properly defined in `mo_ncout_hamocc`), you'll need to add these output fields also to `namelist_definition_blom.xml`.